skeleton/aws/modules/alb/main.tf

```diff
@@ -1,3 +1,7 @@
+locals {
+  enable_stickiness = false
+}
```

Comment:
I'm wondering why using local (1) is better than using a variable with default value (2) in this case. Both approaches does not require supplying the value for this setting when this module is consumed elsewhere, e.g. when we use this module in the `/core/main.tf`.
But the (2) approach has more flexibility since we can override this setting if needed right from the `core/main.tf` without changing the code of this `alb` module.

Reply:
From my understanding: 
- locals make the values more explicit as the value is in the same file as the resource declaration
- variable is needed if we want to have a "long-term" different values between environments. 

For stickiness, I would relly discourage any project to have a different config between environments 🤔

Reply:
that makes sense